en/telemetry/sik_radio.md

```diff
@@ -0,0 +1,71 @@
+# SiK Radio
+
+[SiK radio](https://github.com/LorenzMeier/SiK) is a collection of firmware and tools for telemetry radios.
+
+Hardware for the SiK radio can be obtained from various manufacturers/stores in variants that support different range and form factors. Typically you will need a pair of devices - one for the vehicle and one for the ground station. The ground station-based radio is connected via USB (essentially plug-n-play). The vehicle is connected to the flight-controller's `TELEM` port, and typically requires no further configuration.
```

Comment:
@bkueng I think this is correct - can you please confirm? Especially that you connect to TELEM and that you don't need to do further configuration in PX4.

Reply:
This is correct. I think I'd write `TELEM1`, since most boards provide several telemetry outputs, and the first one is used for the ground station.